libreyolo/models/rfdetr/loss.py

```
@@ -504,58 +515,36 @@ def loss_masks(self, outputs, targets, indices, num_boxes):
-        norm_tgt_xy = tgt_xy / area_scale[:, None, None]
-        loss_l1 = (F.l1_loss(norm_pred_xy, norm_tgt_xy, reduction="none").sum(-1) * visible_f).sum()
-        loss_l1 = loss_l1 / visible_count.sum().clamp_min(1.0)
-
-        sigmas = self.oks_sigmas.to(device=src_keypoints.device, dtype=src_keypoints.dtype)
-        dist2 = ((pred_xy - tgt_xy) ** 2).sum(dim=-1)
-        denom = 2.0 * area[:, None] * (2.0 * sigmas[None, :]).pow(2) + 1e-9
-        oks_loss = 1.0 - torch.exp(-dist2 / denom)
-        per_instance_oks = (oks_loss * visible_f).sum(dim=1) / visible_count
-        per_instance_oks = torch.where(
-            visible.any(dim=1), per_instance_oks, per_instance_oks * 0.0
+        target_keypoints = torch.cat([target["keypoints"][j] for target, (_, j) in zip(targets, indices)], dim=0)
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)